boot-qemu.sh

```diff
@@ -238,6 +226,18 @@ function setup_qemu_args() {
             esac
             ;;
     esac
+    if ${USE_CBL_QEMU:-false} && [[ ${ARCH} = "riscv" || ${ARCH} = "s390" ]]; then
+        QEMU_BINARIES=${BASE}/qemu-binaries
+
+        green "Downloading or updating qemu-binaries..."
+        [[ -d ${QEMU_BINARIES} ]] || git clone https://github.com/ClangBuiltLinux/qemu-binaries "${QEMU_BINARIES}"
+        git -C "${QEMU_BINARIES}" pull --rebase
```

Comment:
consider shallow clone and `git reset --hard FETCH_HEAD`? Though probably overkill as we only have like 2 commits in that repo...it is of binaries though.